gradient/api_sdk/clients/job_client.py

```diff
@@ -75,7 +76,7 @@ def create(
                 name='Example job',
                 command='pip install -r requirements.txt && python mnist.py',
                 ports='5000:5000',
-                job_env={
+                env_vars={
                     'CUSTOM_ENV'='Some value that will be set as system environment',
```

Comment:
nit: this isn't valid python, we should change it to something like (or whatever):

```
env_vars={'CUSTOM_ENV': 'my value'}
```